lib/search/Search.rocketmap_mad.php

```diff
@@ -146,19 +167,33 @@ public function search_portals($lat, $lon, $term)
 
     public function search($dbname, $lat, $lon, $term)
     {
-        global $db, $defaultUnit, $maxSearchResults;
+        global $db, $defaultUnit, $maxSearchResults, $noBoundaries, $boundaries;
+
+        if (!$noBoundaries) {
+            $coords = " AND (ST_WITHIN(point(lat,lon),ST_GEOMFROMTEXT('POLYGON(( " . $boundaries . " ))'))) ";
```

Comment:
Mad uses latitude and longitude for pokestops and gyms. Also: gyms require to be g.latitude and g.longitude coz of the left join

Reply:
Their SELECT statements have both lat/latitude and lon/longitude. So lat/lon should work:
"SELECT pokestop_id,name,latitude AS lat,longitude AS lon,"
"SELECT g.gym_id,gd.name,g.latitude AS lat,g.longitude AS lon,"

Reply:
Hart sent me the structure of MAD's DB so I could test these two queries. I did have to change to latitude/longitude.